components/dashboard/src/start/IntermediatePrebuildStatus.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import StatusRunning from "../icons/StatusRunning.svg";
+
+function IntermediatePrebuildStatus() {
```

Comment:
To show the prebuild status, we need to find prebuilds related to a project. When showing the logs in the prebuild detail view, we already have the project information. This is not the case during workspace start. However, we do have a prebuild ID and status. 
This intermediate status component is a workaround so that we can show the prebuild status "Running" while waiting for the prebuild updates. Once the prebuild updates are received and we have a prebuild object, we show those accordingly.

Reply:
We don't need this intermediate status anymore because we are now fetching the prebuild from the beginning.